### ISSUE TITLE
(Temp NJP site, but also will apply to BrooklynKTheme) Lucky orange tracking is not working for starting checkout or placing order

### ISSUE THREAD

**Issue by AlanDuong07** [AlanDuong07/NJPBrooklynKTheme#49]
I'm suspecting this is because the lucky orange code is not being generated since its in the header. And i'm pretty sure the header is not the same once checkout begins.

**Comment by AlanDuong07** [comment]
I figured out the issue: the tracking code is set in the NJP backend to display on the <head> of the html, as a script of course. But in the checkout page, the header (and footer) changes visually and in the backend as well. The script is not being generated in the header on checkout. I may have to manually put it there myself. I also need to remember to make this change once the BrooklynK theme is live.

**Comment by AlanDuong07** [comment]
Fixed it! I did a bunch of investigating, and even looked at the other Big Commerce site (eDiscountPerfumes), and found out that tracking codes can go in another place: Advanced Store Settings -> Data Solutions -> Google analytics. In fact, that is where the tracking codes are for Bing and Google. So I went into NJP's backend and deleted the script from Script manager and instead put it into the Google Analytics section. For some reason, the tracking code is persistent even in the checkout page when it is in that section, which is exactly what I want. 

And also, now when I upload the BrooklynK theme, the code will still be present there. So no need to worry about that :)
